frontend/ui/quickstart.lua

```diff
@@ -0,0 +1,70 @@
+--[[--This module is responsible for generating the quickstart guide.
+]]
+local DataStorage = require("datastorage")
+local FileConverter = require("apps/filemanager/filemanagerconverter")
+local gettext = require("gettext")
+local lfs = require("libs/libkoreader-lfs")
+local T = require("ffi/util").template
+
+local quickstart_shown = G_reader_settings:readSetting("quickstart_shown") or nil
+local quickstart_shown_version = G_reader_settings:readSetting("quickstart_shown_version") or nil
+local quickstart_force_show_version = 201511982
+local language = G_reader_settings:readSetting("language") or "en"
+local version, _, rev = G_reader_settings:current_version()
+
+local quickstart_guide = T(gettext([[
+# KOReader Quickstart Guide
+
+Welcome to KOreader. You can activate the menu by swiping down from the top of the screen. Clicking outside the menu or swiping up on the menu will discard it. Turning pages can be done either by swiping left and right or by single taps on the left or right side of the screen.
+
+**Main menu**
+
+![Menu](../resources/icons/menu-icon.png) You can always view this quickstart guide again through *Help* → *Quickstart guide* in the top right menu.
+
+**Settings**
+
+![Settings](../resources/icons/appbar.settings.png) You can change the language and other settings through the gear icon.
+
+------------
+
+Generated by KOReader %1.
+]]),
+    rev)
```

Comment:
The screenshot is actually slightly outdated, sorry about that. As I had to do the int version for comparison I included it in the document and then decided it'd be better to just stick with the regular versioning scheme all around.
![screenshot_2017-04-14_09-09-31](https://cloud.githubusercontent.com/assets/202757/25036112/2978c754-20f2-11e7-8a68-f2373a3f7c10.png)